tests/unittests/unittest.cpp

```diff
@@ -1,23 +1,58 @@
 // STL
+#include "catch2/catch_test_macros.hpp"
+#include "catch2/matchers/catch_matchers_string.hpp"
+#include "catch2/matchers/catch_matchers_vector.hpp"
+
+#include <catch2/matchers/catch_matchers.hpp>
+#include <nlohmann/detail/iterators/iter_impl.hpp>
+#include <nlohmann/detail/json_ref.hpp>
```

Comment:
Should we be including `detail` headers from a library? Can we make IWYU happy with including only the `nlohmann/json.hpp` (or maybe even the `json_fwd.hpp`) header?

Reply:
In fact looking at what iwyu pulls in here, I would actually be fine with simply including the tests.

Reply:
Only `json_fwd.hpp` is being included.